plugins/mongodb/mongodb.ecllib

```diff
@@ -0,0 +1,26 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+EXPORT Language := SERVICE : plugin('mongodbembed')
+  boolean getEmbedContext():cpp,pure,namespace='mongodbembed',entrypoint='getEmbedContext',prototype='IEmbedContext* getEmbedContext()';
+END;
+
+EXPORT getEmbedContext := Language.getEmbedContext;
+EXPORT boolean supportsImport := false;
+EXPORT boolean supportsScript := true;
+
+EXPORT updateRecord := {INTEGER matched_count, INTEGER modified_count};
```

Comment:
Recommend renaming 'updateRecord' for clarity (it currently sounds like a verb).  This is, in this case, a result of call; maybe use 'updateRecordResult' ?